install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -46,6 +46,11 @@ initContainers:
     privileged: true
     {{- end }}
   restartPolicy: Always
+  env:
+  {{- if (isset .ObjectMeta.Annotations `sidecar.istio.io/inboundInterceptionSplit`) }}
```

Comment:
- name is not very good - 'split' is an implementation detail. But not sure what is a better name we can leave it.
- whatever it is - should match whatever is set in Values.global. 

The annotations should only be used to override settings defined in mesh configuration (values.yaml).

Reply:
My point is to not set port in values.yaml. 
That's the config user sets - they should not set the port. 

How about: 
global:
   proxy:
      includeInboundPorts: *

( the feature is actually controlling capture of all ports instead of using containerPorts ).

Reply:
will hard code this port for now
and 
override the unimplemented `includeInboundPorts`. Notes that `includeInboundPorts` is not `sidecar.istio.io/includeInboundPorts`